curriculum/challenges/_meta/legacy-full-stack-certificate/meta.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Full Stack Certificate",
-  "dashedName": "full-stack-certificate",
+  "name": "Legacy Full Stack Certificate",
+  "dashedName": "legacy-full-stack-certificate",
```

Comment:
I know we have some `legacy-x-certificates` already, but shouldn't this remain `full-stack-certificate` forever?  Only the text should be changed (like "name" and "title"), right?

Reply:
That is correct.

Reply:
Yes, thanks for pointing that out. It doesn't seem to affect anything currently, but will make things easier to sort out in the future.

It's strange, but I played around with `"name"` and `"dashedName"` here. It seems like `"name"` controls the cert's `block` flag in the DB. Then `title` in `/curriculum/challenges/english/12-certificates/legacy-full-stack-certificate/legacy-full-stack-certificate.md` sets both `title` and `dashed-name` for the cert in the database.

That makes sense given how the other challenges are handled, though maybe something to keep in mind for generating a single source of truth for the certs client side.